backend/devices/bitbox02/device.go

```diff
@@ -454,6 +454,24 @@ func (device *Device) ListBackups() ([]*Backup, error) {
 	return backups, nil
 }
 
+// CheckBackup checks the integrity of backups and if they are restorable
```

Comment:
the comment is wrong. The call checks whether any of the backups on the sd card is a backup of the current seed on the device .

Reply:
done